spec/models/guest_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Guest, type: :model do
+ it { should have_many :rooms }
```

Comment:
There are 2 associations in your `Guest` model, only one of them is being tested here. We need to also test for the one-to-many association between a Guest and GuestRooms. I also don't see the `#through` relationship test for `:rooms`.

```ruby
it { should have_many :guest_rooms }
it { should have_many(:rooms).through(:guest_rooms) }
```